client/state/products-list/selectors.js

```diff
@@ -33,3 +44,40 @@ export function getProductDisplayCost( state, productSlug ) {
 
 	return product.cost_display;
 }
+
+export const getDiscountedOrRegularPrice = ( state, siteId, plan, isMonthlyPreference ) => {
```

Comment:
instead of passing a boolean parameter here (which is quite opaque when reading the code) could we consider one of two alternatives?

 - create two separate functions to expose as the API for it, like `getMonthlyDiscountedOrRegularPrice`
 - pass a string or constant instead which can remove the ambiguity `getDiscountedOrRegularPrice( …, 'MONTHLY_PRICE' )`

also, something seems to stand out that this function is named with an `or` - could it be that we're conflating two different things into one or could it be that we're making the name more complicated than it needs to be? what makes this different than just `getPrice()`?

Reply:
I agree about the general code style! In this particular case though, this is a wrapper over already existing API which leaves us with either making it inconsistent or copying&pasting the function just to pass a different boolean flag.

Good note about the name - I renamed it to `getPlanPrice`.